### PR TITLE
Yolact docs

### PR DESCRIPTION
**Pull request by dbogunowicz**
This PR:
- Creates yolact doc in the root directory of the integration

**Test Plan:**
_Manual_

- [x] - Checked the deepsparse-related commands
- [x] -  Checked the sparseml-related commands

_Automated_
N/A

### CODE REVIEW

File: src/deepsparse/transformers/eval_downstream.py [hunk 1]
```diff
@@ -53,8 +53,8 @@
     --dataset squad
 """
 
-
 import argparse
+import json
```
Comment (bfineran) [inline src/deepsparse/transformers/eval_downstream.py:57]: bad rebase?

File: src/deepsparse/yolact/README.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+Below we describe two possibilities to obtain the required ONNX model.
+
+### Exporting the ONNX File From the Contents of a Local Directory
+This pathway is relevant if you intend to deploy a model created using the [SparseML](https://github.com/neuralmagic/sparseml) library. 
+For more information refer to the [appropriate YOLACT integration documentation in SparseML](...).
+
+After training your model with `SparseML`, locate the `.pth` file for the model you'd like to export and run the `SparseML` integrated YOLOv5 ONNX export script below.
+
+```bash
+# TODO
```
Comment (bfineran) [inline src/deepsparse/yolact/README.md:35]: reminder: TODO

File: src/deepsparse/yolact/README.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+endpoint descriptions and support for making sample requests.
+
+An example of starting and requesting a DeepSparse Server for YOLOv5 is given below.
+
+#### Installation
+The Deepsparse Server requirements can be installed by specifying the `server` extra dependency when installing
+DeepSparse.
+
+```bash
+pip install deepsparse[yolo,server]
```
Comment (bfineran) [inline src/deepsparse/yolact/README.md:68]: nit: yolo

File: src/deepsparse/yolact/README.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+<img src="sample_images/thailand.jpg" alt="original" width="400"/> <img src="sample_images/thailand-annotated.jpg" alt="annotated" width="400"/>
+</p>
+<p align = "center">
+Image annotated with 82.8% sparse and quantized YOLACT
+</p>
+
+If a `--model_filepath` arg isn't provided, then `zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none` will be used by default.
+
+
+#### HTTP Server (TODO)
```
Comment (bfineran) [inline src/deepsparse/yolact/README.md:120]: reminder: TODO

File: src/deepsparse/yolact/README.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+You can quickly do benchmarking tests on your own with a single CLI command!
+
+You only need to provide the model path of a SparseZoo ONNX model or your own local ONNX model to get started:
+
+```bash
+deepsparse.benchmark \
+    zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none \
+    --scenario sync 
+
+>> Original Model Path: zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none
```
Comment (bfineran) [inline src/deepsparse/yolact/README.md:153]: add hardware and deepsparse version this benchmark was run on

**Comment by rahul-tuli** [comment]
Maybe changes added to `utils.py` belong in a separate PR?

**Comment by dbogunowicz** [comment]
I'd say let's land #500 and the potential future PRs that inherit from this one prior to landing this PR.

**Comment by dbogunowicz** [comment]
> Maybe changes added to `utils.py` belong in a separate PR?

Yeah, they prolly do. Let me take it out from the PR and put it into the original yolact PR.

**Review by bfineran — APPROVED** [review]

File: src/deepsparse/yolact/README.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+<img src="sample_images/thailand.jpg" alt="original" width="400"/> <img src="sample_images/thailand-annotated.jpg" alt="annotated" width="400"/>
+</p>
+<p align = "center">
+Image annotated with 82.8% sparse and quantized YOLACT
+</p>
+
+If a `--model_filepath` arg isn't provided, then `zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none` will be used by default.
+
+
+#### HTTP Server
```
Comment (corey-nm) [inline src/deepsparse/yolact/README.md:120]: Whats your sense for the most common deployment users do? Is server pretty common use case? Wondering if this HTTP Server section should be the first example under Deployment Examples (cause its super easy to try out!)?

Like 
```
## Deployment examples
<http server>
#### benchmarking
#### annotate script for testing
#### using the pipeline in code
```
?

**Review by corey-nm — APPROVED** [review]
Looks good! Left some suggestions but nothing blocking

File: src/deepsparse/yolact/README.md (hunk 6 above)
Comment (bfineran) [reply src/deepsparse/yolact/README.md:120]: the annotate cli is the most popular use case for the object detection flows - good call out though

**Review by bfineran — APPROVED** [review]

**Review by corey-nm — APPROVED** [review]

**Pull request merged**